bindings/nodejs/README.md

```diff
@@ -3,5 +3,5 @@
 ## Build
 
 ```shell
-pnpm run build:debug
+yarn run build:debug
```

Comment:
**question**: Why change pnpm into `yarn`?

Reply:
This package manager is specified by the latest CLI. I regenerated the project.
However, I haven't delved into the root cause yet.

Reply:
Understood. Let us continue to utilize `pnpm` and eliminate all content related to `yarn`.

Reply:
Reason found.  https://github.com/napi-rs/napi-rs/issues/1428#issuecomment-1378365197
We may keep this.

Reply:
We don't need cross compile so far. Let's keep as-is.